spec/fixtures/v4_mixed_sync_async_interactions.json

```diff
@@ -0,0 +1,77 @@
+{
+    "consumer": {
+        "name": "Consumer"
+    },
+    "provider": {
+        "name": "Provider"
+    },
+    "interactions": [
+        {
+            "description": "a message interaction",
+            "providerStates": [
+                {
+                    "name": "provider is at state one",
+                    "params": {}
+                },
+                {
+                    "name": "provider is at state two",
+                    "params": {}
+                }
+            ],
+            "contents": {
+                "some": "content"
+            },
+            "metadata": {
+                "meta": "data"
+            }
+        },
+        {
+            "description": "a request for alligators",
+            "providerState": "alligators exist",
+            "request": {
+                "method": "get",
+                "path": "/alligators"
+            },
+            "response": {
+                "status": 200,
+                "body": {
+                    "alligators": [
+                        {
+                            "name": "Bob",
+                            "phoneNumber": {
+                                "json_class": "Pact::Term",
+                                "data": {
```

Comment:
Where'd this example come from? It looks like a very old (v1 or earlier) format that uses Ruby's HTTP matcher serialisation (i.e. it shouldn't be in the pact file).